configs/model/fre.yaml

```
@@ -0,0 +1,13 @@
+model:
+  class_path: anomalib.models.Fre
+  init_args:
+    backbone: resnet50
+    layer: layer3
+    pre_trained: true
+    pooling_kernel_size: 2
+    input_dim: 65536
+    latent_dim: 220
+    score_type: fre
+
+trainer:
+  max_epochs: 6
```

I see that you set this to 220 in the model definition. it would be good to add it here for those who are using the CLI
```suggestion
  max_epochs: 220
```